content_creator.py

```diff
@@ -146,9 +148,23 @@ def copy_test_files(bundle_test):
             shutil.copyfile(path, os.path.join(bundle_test, os.path.basename(path)))
 
 
-def main(circle_artifacts):
-    print('Starting to create content artifact...')
+def update_content_version(key, new_value):
+    f = open('./Scripts/CommonServerPython/CommonServerPython.py', 'r')
+    lines = f.readlines()
+    f.close()
+    for i, line in enumerate(lines):
```

Comment:
should either use regex over file content here or read the yml into a dict and pull the relevant key